homeassistant/components/homekit_controller/media_player.py

```diff
@@ -16,15 +16,10 @@
     MediaPlayerDeviceClass,
     MediaPlayerEntity,
     MediaPlayerEntityFeature,
+    MediaPlayerState,
 )
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import (
-    STATE_IDLE,
-    STATE_OK,
-    STATE_PAUSED,
-    STATE_PLAYING,
-    STATE_PROBLEM,
-)
+from homeassistant.const import STATE_OK, STATE_PROBLEM
```

Comment:
Do we allow custom states for media_player?

Reply:
No, this needs to be addressed.

Reply:
Thanks for confirming. I have adjusted to use MediaPlayerState.ON and MediaPlayerState.OFF instead.